include/SetupDialog.h

```diff
@@ -209,9 +209,9 @@ private slots:
 	QLineEdit * m_themeDirLineEdit;
 	QLineEdit * m_ladspaDirLineEdit;
 	QLineEdit * m_gigDirLineEdit;
-	QLineEdit * m_sf2DirLineEdit;
+	QLineEdit * m_soundFontDirLineEdit;
```

Comment:
(Non-blocking) Per the [Coding Conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions#pointers):

> The `*` symbol in pointer declarations MUST be adjacent to the type, not the variable name. This rule also applies to `**`, `&`.

```suggestion
	QLineEdit* m_soundFontDirLineEdit;
```